### PR TITLE
ENCD-4503 micro RNA qc

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

**Comment by jahilton** [comment]
Many changed files don't end with an empty line

**Comment by jahilton** [comment]
Is there a specific reason why assay_term_name wasn't added as a facet to schemas/micro_rna_quality_metric.json?

File: src/encoded/schemas/micro_rna_quantification_quality_metric.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "expressed_mirnas": {
+            "title": "Expressed miRNAs",
+            "description": "Number of miRNAs expressed",
+            "comment": "miRNAs expressed < 2 cpms are filtered",
```
Comment (ottojolanki) [inline src/encoded/schemas/micro_rna_quantification_quality_metric.json]: If this comment is needed, I think come clarification may be good. "miRNAs expressed < 2 cpms are not counted as expressed" or something like that.

**Comment by forresttanaka** [comment]
The e828ec88-8f74-437f-8bf7-8535ee34819e experiment in src/encoded/tests/data/inserts/experiment.json has an extra comma at the end of the last property that causes an error during local instance indexing (and I expect Travis CI too). The extra comma prevents experiments from indexing, and causes validation errors in objects referring to them.

**Pull request merged**